### ISSUE TITLE
Color contrast issue for Primary Action

### ISSUE THREAD

**Issue by coryrylan** [vmware/clarity#4225]
## Describe the bug

In the v3 branch the background of the primary action no longer passes AA contrast for any text size. This is used on the solid primary action button as well as a few other places.

In v2 the primary button color is `hsl(201, 100%, 36%)` while in v3 it is `hsl(198, 100%, 38%)` which fails AA.

## How to reproduce

Compare the background color of the solid button
https://v2.clarity.design/buttons
https://clarity.design/documentation/buttons

https://contrast-ratio.com/#hsl%28201%2C%20100%25%2C%2036%25%29-on-%23fff
https://contrast-ratio.com/#hsl%28198%2C%20100%25%2C%2038%25%29-on-%23fff

## Expected behavior

Primary color should pass AA for any text size.

**Comment by chlane** [comment]
Fixed verified on https://clarity.design/documentation/buttons. #fff against #0072a3 yields a highly compliant contrast ratio of 5.34:1 as seen at https://www.levelaccess.com/color-contrast-checker/.  The relevant CSS is below.

.btn-primary .btn, .btn.btn-primary {
    border-color: #0072a3;
    border-color: var(--clr-btn-primary-border-color,#0072a3);
    background-color: #0072a3;
    background-color: var(--clr-btn-primary-bg-color,#0072a3);
}
